### PR TITLE
Always navigate to day view when action is navigate.DATE

### PR DESCRIPTION
**Pull request by tobiasandersen**
Fix for #202

### CODE REVIEW

**Comment by jquense** [comment]
thanks!

**Comment by MarshallOfSound** [comment]
I came here to try to remove this behavior.  Now when you are on month view and you send a custom navigate request to a specific date we always end up on the day view (which is not expected).  Would you accept a PR @jquense to add a flag so that this day view change can be explicitly prevented?

**Comment by jquense** [comment]
@MarshallOfSound how are you doing: " send a custom navigate request". Passing a new date to the Calendar won't hit this code.

File: src/Calendar.js [hunk 1]
```diff
@@ -518,15 +518,7 @@ let Calendar = React.createClass({
     onNavigate(date, view)
 
     if (action === navigate.DATE)
-      this._viewNavigate(date)
-  },
-
-  _viewNavigate(nextDate) {
-    let { view, date, culture } = this.props;
-
-    if (dates.eq(date, nextDate, view, localizer.startOfWeek(culture))) {
```
Comment (jquense) [inline src/Calendar.js]: @tobiasandersen I remember why this check was here. If you try to navigate to an arbitrary date from the month view you can't now without it also switching to the day view

Comment (tobiasandersen) [inline src/Calendar.js]: Oh, I see. What do you want to do with it?

Comment (jquense) [inline src/Calendar.js]: i'm working on something. the problem is conflating these two things in the first place, but i need to add some more props to fix it for good :P I'll have something up in a bit

**Pull request merged**